module/zfs/spa.c

```diff
@@ -2491,17 +2491,84 @@ spa_activity_check_required(spa_t *spa, uberblock_t *ub, nvlist_t *label,
 	return (B_TRUE);
 }
 
+static uint64_t
```

Comment:
Can you add a comment describing the return value of this function?  Is it the number of nanoseconds to wait for a good MMP write?

Reply:
Yes, number of nanoseconds to wait and test for an uberblock change before deciding it is safe to import.

Reply:
Done